docs/common/istio-ambient-update.adoc

```diff
@@ -0,0 +1,653 @@
+// Variables embedded for GitHub compatibility
+:istio_latest_version: 1.26.3
+:istio_latest_version_revision_format: 1-26-3
+:istio_latest_tag: v1.26-latest
+:istio_release_name: release-1.26
+:istio_latest_minus_one_version: 1.26.2
+:istio_latest_minus_one_version_revision_format: 1-26-2
+
+link:../README.adoc[Return to Project Root]
+
+[[updating-istio-in-ambient-mode]]
+== Updating Istio in Ambient Mode
+
+This guide provides procedures for updating Istio when deployed in ambient mode using the Sail Operator. The upgrade process includes updating the Istio control plane, IstioCNI, and ZTunnel components.
+
+For general information about update strategies (InPlace and RevisionBased) and versioning, see link:../update-strategy/update-strategy.adoc#update-strategy[Update Strategy].
+
+For information about installing Istio in ambient mode, see link:./istio-ambient-mode.adoc#introduction-to-istio-ambient-mode[Istio Ambient Mode].
+
+*Note:* Istio's ambient mode is available starting from Istio version 1.24.0.
+
+*Key differences in ambient mode:*
+
+* Updates involve three components: Control Plane, IstioCNI, and ZTunnel
+* Components must be updated in sequence: Control Plane → IstioCNI → ZTunnel
+* No pod restart needed for workloads (workloads automatically connect to new control plane)
```

Comment:
I think we can remove this statement as technically workloads do not connect to the control plane, the ztunnel node agent connects to the control plane.

```suggestion
* No pod restart needed for workloads
```

Reply:
Addressed on 35ec7b84